src/Command/BaseMigrationCommand.php

```diff
@@ -11,14 +11,12 @@
 use Spiral\Migrations\State;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Output\OutputInterface;
-use Yiisoft\Yii\Cycle\Helper\CycleOrmHelper;
 
 abstract class BaseMigrationCommand extends Command
 {
     protected DatabaseManager $dbal;
     protected MigrationConfig $config;
     protected Migrator $migrator;
-    protected CycleOrmHelper $cycleOrmHelper;
 
     protected static $migrationStatus = [
```

Comment:
Is it meant to be extended? Could it be constant? Also `array` could be specified as type.